src/toolchain/export-web.md

```diff
@@ -0,0 +1,155 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more difficult to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
+However, setting up a web server to host and share your game is considered out of scope of this guide, and is best explained elsewhere.
+
+Information to consider before starting:
+
+```admonish warning
+Web support with `gdext` is experimental and should be understood as such before proceeding.
+```
+
+---
+
+
+## Installing Pre-Requisites
+
+Install a nightly build of `rustc`, the `wasm32-unknown-emscripten` target for `rustc`, and `rust-src`.
+The reason that a nightly `rustc` is required is that nightly is required to build `std` (`-Zbuild-std`).
+Assuming that Rust was installed with `rustup`, this is quite simple.
+
+  ```sh
+  rustup toolchain install nightly
+  rustup component add rust-src --toolchain nightly
+  rustup target add wasm32-unknown-emscripten --toolchain nightly
+  ```
+
+Next, install Emscripten.  The simplest way to achieve this is to install `emsdk` from the git repo. Recommended to use version 3.1.39 for now[^1]
+
+```sh
+git clone https://github.com/emscripten-core/emsdk.git
+cd emsdk
+./emsdk install 3.1.39
+./emsdk activate 3.1.39
+source ./emsdk.sh     (or ./emsdk.bat on windows)
+```
+
+It would also be HIGHLY recommended to follow the instructions in the terminal to add `emcc`[^2] to your `PATH`.
+If not, it is necessary to manually `source` the `emsdk.sh` file in every new terminal prior to compilation.
+This is platform-specific.
+
+---
+
+
+## Project Configuration
+
+Enable the `experimental-wasm` feature on `gdext` in the `Cargo.toml` file.
```

Comment:
```suggestion
Enable the `experimental-wasm` feature on gdext in the `Cargo.toml` file.
```

gdext is a name, not a symbol (you can't know that, but we don't format it elsewhere, either) 🙂

Reply:
I think I was just going with marking anything technical name/cli ish to mark in backticks like that. I'll remove the other two instances of it for gdext then.